pkg/segment/query/processor/queryprocessor.go

```diff
@@ -74,8 +76,12 @@ func NewQueryProcessor(firstAgg *structs.QueryAggregators, queryInfo *query.Quer
 		firstProcessorAgg = firstProcessorAgg.Next
 	}
 
+	timeOrdered := true
 	dataProcessors := make([]*DataProcessor, 0)
 	for curAgg := firstProcessorAgg; curAgg != nil; curAgg = curAgg.Next {
+		if curAgg.SortExpr != nil {
+			timeOrdered = false
+		}
```

Comment:
I don't think you need this. Eventually you use this in the websocket response, but I think you can always pass `false` for that since we always give results to the frontend in the correct order (and eventually we could remove this parameter from the frontend). The frontend is only using it [here](https://github.com/siglens/siglens/blob/develop/static/js/search.js#L1195-L1200) to set the sort order.

Also, if we do have this calculation for whether it's sorted, we'd have to handle other cases too (like `dedup` with the `sortby` option specified). It will be much simpler if we don't need this.

@sonamgupta21 am I missing any reason we'd still need to set this even if the backend always gives the frontend the records in the correct order?